.github/workflows/test_suite.yml

```diff
@@ -17,24 +17,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        browser: [chrome, firefox]
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
+        browser: [chromium, firefox]
+        python-version: ["3.12", "3.13"]
```

Comment:
Let's keep 3.10 in for now as it'll get security updates for another year

Reply:
Added >=3.10